#- name: "Stop datanode"
#  command: ./hdfs --daemon stop datanode
#  args:
#    chdir: /usr/local/hadoop/bin

# issue here with non-zero exit code?!
- name: "Start datanode"
  command: ./hdfs --daemon start datanode
#  register: verification
#  changed_when: verification.rc == 1
  ignore_errors: yes
  args:
    chdir: /usr/local/hadoop/bin


#/usr/local/hadoop/bin/hdfs --daemon start datanode